rust/src/polars/mod.rs

```diff
@@ -284,16 +271,25 @@ impl Function<DslPlan, ExprPlan> {
             })
         })
     }
+}
 
+impl<TI: 'static> Function<TI, ExprPlan> {
     /// # Proof Definition
-    /// Returns a Function that specifies what the function should return if evaluated on an empty partition.
+    /// Returns a Function that specifies how to impute missing values representing empty partitions.
     ///
     /// Polars only keeps non-empty partitions in group-by,
     /// so this is used to fill missing values after joining with an explicit key set.
     pub(crate) fn fill_with(self, value: Expr) -> Self {
-        Self::new_fallible(move |arg: &DslPlan| {
+        // Without repeat, the expression will be scalar-valued,
+        // and broadcast later to the required length.
+        // The drawback is that randomized plugins, like noise and report_noisy_max,
+        // will then only be applied to one row,
+        // and the one noisy row will then be broadcast to the entire column.
```

Comment:
Perhaps more subjunctive to clarify that this is no longer a problem?
```suggestion
        // Without this repeat, the expression would be scalar-valued,
        // and broadcast later to the required length.
        // This would cause randomized plugins, like noise and report_noisy_max,
        // to only be applied to one row,
        // and the one noisy row would then be broadcast to the entire column.
```